src/Screens/App/Screens/BooksDashboard/Screens/BookList/index.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import Book from './Screens/Book';
+import books from './Resources/books';
+import './style.css';
+
+class BookList extends Component {
+
+  render() {
+    let check = (book) => {
+      if (this.props.filterType.length != 0 && this.props.filter.length != 0) {
```

Comment:
null, undefined and empty string are [falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) in js. here you can: `if (this.props.filterType && this.props.filter) {`